src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -0,0 +1,272 @@
+
+    def write_cont(self, cont):
+        """Write the test data using either ior or the obj API.
+        
+        Args:
+            cont (TestContainer): the container to write to.
+
+        Returns:
+            list: string list of properties from daos command.
+
+        """
+        if cont.type.value == "POSIX":
+            self.run_ior_with_params(
+                "DAOS", "/" + self.test_file,
+                cont.pool, cont, flags=self.ior_flags[0])
+        else:
+            self.dataset_gen(cont, 1, 1, 1, 0, [1024], [])
+
+        # Write the user attributes
+        cont.open()
+        attrs = self.get_cont_usr_attr()
+        cont.container.set_attr(attrs)
+        cont.close()
+
+        # Return existing cont properties
+        return self.get_cont_prop(cont)
+
+    def verify_cont(self, cont, check_attr_prop=True, prop_list=None):
+        """Read-verify test data using either ior or the obj API.
+
+        Args:
+            cont (TestContainer): the container to verify.
+            check_attr_prop (bool, optional): whether to verify user
+                attributes and cont properties. Defaults to False.
+            prop_list (list, optional): list of properties from get_cont_prop.
+                Required when check_attr_prop is True.
+
+        """
+        if cont.type.value == "POSIX":
+            self.run_ior_with_params(
+                "DAOS", "/" + self.test_file,
+                cont.pool, cont, flags=self.ior_flags[1])
+        else:
+            self.dataset_verify(cont, 1, 1, 1, 0, [1024], [])
+
+        if check_attr_prop:
+            cont.open()
+            self.verify_cont_prop(cont, prop_list)
+            self.verify_usr_attr(cont)
+            cont.close()
+
+    def get_cont_prop(self, cont):
+        """Get all container properties with daos command.
+
+        Args:
+            cont (TestContainer): the container to get props of.
+
+        Returns:
+            list: string list of properties from daos command.
+
+        """
+        prop_result = self.daos_cmd.container_get_prop(
+            cont.pool.uuid, cont.uuid)
+        prop_text = prop_result.stdout_text
+        prop_list = prop_text.split('\n')[1:]
+        return prop_list
+
+    def verify_cont_prop(self, cont, prop_list):
+        """Verify container properties against an input list.
+        Expects the container to be open.
+
+        Args:
+            cont (TestContainer): the container to verify.
+            prop_list (list): list of properties from get_cont_prop.
+
+        """
+        actual_list = self.get_cont_prop(cont)
+
+        # Make sure sizes match
+        if len(prop_list) != len(actual_list):
+                self.log.info("Expected\n%s\nbut got\n%s\n",
+                              prop_list, actual_list)
+                self.fail("Container property verification failed.")
+
+        # Make sure each property matches
+        for prop_idx in range(len(actual_list)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len